Documentation/ContentElements/Content/Index.rst

```diff
@@ -1,74 +1,198 @@
+    :class: with-border
 
 Common page icons explained:
 
-#. Use this icon to edit all the content elements of a column.
+#.  Use this icon to edit all the content elements of a column.
+
+#.  Add a new content element to the column at this specific position.
+
+#.  Content element icon. This can provide a visual clue to the type of content
+    element. Click to access the context menu for the element (to perform
+    actions like *copy*).
 
-#. Add a new content element to the column at this specific position.
+#.  This group of icons is enabled when hovering over the whole element.
 
-#. Content element icon. This can provide a visual clue to the type of content
-   element. Click to access the context menu for the element (to perform
-   actions like *copy*).
+    *  Use the pencil icon to edit the element.
+    *  Enable/disable the element to show or hide it in the frontend
+    *  Delete the element.
 
-#. This group of icons is enabled when hovering over the whole element.
+#.  If the edit icons are missing you do not have sufficient permission to
+    edit the content element in question.
+
+..  _content-working-hiding:
+
+Hiding content elements
+=======================
 
-   *  Use the pencil icon to edit the element.
-   *  Enable/disable the element to show or hide it in the frontend
-   *  Delete the element.
+You can temporarily hide content that should be hidden for a while but will be
+needed later. You can also hide a content element while you are still working
+on it. Hidden content elements are not visible in the frontend
 
-.. _content-working-moving:
+..  figure:: /Images/ManualScreenshots/ContentElements/HideContent.png
+    :alt: A content element in the page module is hidden
+
+    Use the "hide" button to hide a content elements. The second content element is hidden and can be unhidden by the same button.
+
+A content element can also be hidden in the "Access" tab of its edit form, its
+context menu or the `List module <https://docs.typo3.org/permalink/t3editors:using-the-list-module-effectively>`_.
+
+..  _content-working-moving:
 
 Moving content elements
 =======================
 
-You can move content elements using drag and drop.
+You can move content elements using drag and drop. You can either drop them
+in one of the highlighted areas on the current page or drop them on a different
+page within the page tree.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/MoveContentElement.png
+    :alt: Screenshot of moving a content element via drag and drop in the TYPO3 backend
+
+    Drop the content element on a page in the page tree to move it to another page
+
+..  _content-working-copy-paste:
+
+Copy and page content elements
+==============================
+
+..  figure:: /Images/ManualScreenshots/ContentElements/CopyContent.png
+    :alt: The context menu of a content menu in the TYO3 backend
+
+    Use the context menu to copy a content element
```

Comment:
There is no TYPO3 context menu on the content element. Only the one from browser will be shown. Do you mean the context menu when clicking on the upper left icon of an content element?
```suggestion
    Use the context menu of the upper left button of the content element to start copying a content element
```